NEWS.md

```diff
@@ -2,6 +2,10 @@
 
 ## Version 0.5.3 (2024-03-06)
 
+### Bugfix
```

Comment:
This is not a change coming in in 0.5.3 as the next release, in which it will be included, will be 0.5.4. In this case, we have to write it as "Unversioned". I will however fix it in the next version.